website/client/js/services/costumeServices.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Costume', costumeFactory);
+
+  costumeFactory.$inject = [
+    'Content'
+  ];
+
+  function costumeFactory(Content) {
+    function animalInfo(name, type) {
+      var tmp = name.split('-');
+      var data = {
+        egg: tmp[0],
+        potion: tmp[1]
+      };
+      data.isBasic = Boolean(Content.hatchingPotions[data.potion]);
```

Comment:
Okay, just reading the `isBasic` key, I _thought_ this would refer to the drop pets only. Looking at the actual logic, it looks like this is really checking if this is a special animal or not. So instead, let's do:

```js
data.isSpecial = !Content.hatchingPotions[data.potion];
```